__tests__/main.test.ts

```diff
@@ -581,3 +581,94 @@ test('stale label should be removed if a comment was added to a stale issue', as
   expect(processor.staleIssues.length).toEqual(0);
   expect(processor.removedLabelIssues.length).toEqual(1);
 });
+
+test('stale label should not be removed if a comment was added by the bot (and the issue should be closed)', async () => {
+  github.context.actor = 'abot';
```

Comment:
```suggestion
  octomock.context.actor = 'abot';
```

The `github` object was replaced with `octomock`. Committing this suggestion will cause another error, the `context` object field is __undefined__ at this point.